src/mame/drivers/brother_ax145.cpp

```diff
@@ -0,0 +1,352 @@
+// license:BSD-3-Clause
+// copyright-holders:Bartman/Abyss
+
+#include "emu.h"
+#include "emupal.h"
+#include "screen.h"
+#include "speaker.h"
+#include "machine/timer.h"
+#include "cpu/z180/z180.h"
+#include "debug/debugcpu.h"
+#include "sound/beep.h"
+#include "video/hd44780.h"
+
+#pragma region(AX-145)
```

Comment:
Please don’t use weird pragmas like this

Reply:
ok. removed.